src/core_plugins/metrics/common/calculate_label.js

```diff
@@ -62,8 +77,14 @@ export default function calculateLabel(metric, metrics) {
         additionalLabel += ` (${matches[1]})`;
```

Comment:
Since this text contains parenthesis, we should instead split up the below i18n string into `tsvb.calculateLabel.lookupMetricTypeOfTarget` and `tsvb.calculateLabel.lookupMetricTypeOfTargetWithAdditionalLabel` depending on whether we have set this value here, and instead just set the value to the matches[1] string itself without parantheses, so they can be part of the translated string.